cookbook_test.go

```diff
@@ -6,10 +6,67 @@ import (
 	"net/http"
 	//"os"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 const cookbookListResponseFile = "test/cookbooks_response.json"
-const cookbookTestFile = "test/cookbook.json"
```

Comment:
This constant wasn't in use so I assumed it was supposed to be used as a cookbook response. If it wasn't then I can create a new constant. 👍